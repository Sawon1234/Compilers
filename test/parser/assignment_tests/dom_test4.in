class Iterable <E> () extends Thing { }