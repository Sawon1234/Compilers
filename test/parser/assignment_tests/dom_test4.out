class Iterable < E > ( ) extends Thing { super ( ) ; }